misc/setup.ts

```diff
@@ -8,7 +8,7 @@ import * as dotenv from "dotenv";
 const { provider, loadFixture } = waffle;
 
 import TellorPlayground from "usingtellor/artifacts/contracts/TellorPlayground.sol/TellorPlayground.json";
-import { ERC20 } from "../typechain";
+// import { ERC20 } from "../typechain";
```

Comment:
```suggestion
```

Reply:
Yes, @shreyaspapi, that line can be removed.